girder/utility/webroot.mako

```diff
@@ -24,7 +24,8 @@
                 el: 'body',
                 parentView: null,
                 contactEmail: '${contactEmail | h}',
-                brandName: '${brandName | h}'
+                brandName: '${brandName | h}',
+                bannerColor: '${bannerColor}'
```

Comment:
Changes in #2325 are causing merge conflicts now. Also, all edits to `webroot.mako` should now use the new conventions established by that PR.

Reply:
Yes of course, do you think the `bannerColor` need to be escape in the same way ?